src/ocl/convolutionocl.cpp

```diff
@@ -260,6 +270,7 @@ void ConvolutionDescriptor::FindConvFwdAlgorithm(Handle& handle,
                                                  size_t workSpaceSize,
                                                  bool exhaustiveSearch) const
 {
+    ValidateWorkspace(workSpace, workSpaceSize);
```

Comment:
Let's move this after logging everywhere.

Reply:
done